### PR TITLE
use small datasets

### PR DESCRIPTION
**Pull request by cosunae**
## Purpose

Change the default datasets used for testing to a small dataset.

## Code changes:

- Change default dataset to a small domain size.
- Some products (like radar) still require the entire domain size.

## Checklist
Before submitting this PR, please make sure:

- [x] You have followed the coding standards guidelines established at [Code Review Checklist](https://meteoswiss.atlassian.net/wiki/spaces/UA/pages/20157433/Factory).
- [x] Docstrings and type hints are added to new and updated routines, as appropriate
- [x] All relevant documentation has been updated or added (e.g. README)
- [x] Unit tests are added or updated for non-operator code
- [ ] New operators are properly tested

Additionally, if the PR updates the version of the package

- [ ] The new version is properly set
- [ ] A Tag will be created after the bump

## Review
For the review process follow the guidelines at [Checklist](https://meteoswiss.atlassian.net/wiki/spaces/UA/pages/20156488/Code+Review)

### CODE REVIEW

**Review by twicki — COMMENTED** [review]
Definetly a step in the right direction. I think we can simplify the fixtures a bit, see here: https://github.com/MeteoSwiss-APN/icon_data_processing_incubator/pull/90

File: tests/test_idpi/test_ninjo_k2th.py [hunk 1]
```diff
@@ -36,7 +36,7 @@ def test_ninjo_k2th(data_dir, fieldextra):
     assert_allclose(
         fs_ds["POT_VORTIC_AT_THETA"],
         observed_at_theta["pot_vortic"],
-        atol=1e-9,
+        atol=1e-5,
```
[the remark this answers is not in the document]
Comment (cfkanesan) [reply tests/test_idpi/test_ninjo_k2th.py:39]: I'm not sure why this was necessary. Especially that 1e-5 is at the order of magnitude of the observable signal in the pot vortic field.

[the remark this answers is not in the document]
Comment (cfkanesan) [reply tests/test_idpi/test_ninjo_k2th.py:39]: Could be maybe an edge effect due to the differential operators and the reduced computational domain.

[the remark this answers is not in the document]
Comment (cfkanesan) [reply tests/test_idpi/test_ninjo_k2th.py]: Considering that the expected values are in the 1e-5 range, it doesn't really make sense to me to put in an absolute tolerance of 1e-5. That would be comparable to setting an atol of 1e5 for the pressure field. The value for the rtol is fine in my opinion.

**Review by cfkanesan — APPROVED** [review]

**Pull request closed without merging**